packages/services/api/src/modules/operations/providers/operations-reader.ts

```diff
@@ -1127,34 +1143,47 @@ export class OperationsReader {
         queryId: '_getTopOperationsForSchemaCoordinates',
         query: sql`
           WITH "top_operations_by_coordinates" AS (
-            SELECT
+            SELECT DISTINCT
               "coordinates_daily"."coordinate" AS "coordinate",
               "coordinates_daily"."hash" AS "hash",
-              SUM("coordinates_daily"."total") AS "total"
+              "operations_join"."total" AS "total"
             FROM
```

Comment:
We selected the wrong total count here... This caused bugs where the percentage of traffic is calculated above 100% if two different clients send the same query (identical hash), but one client is excluded from the breaking change detection.